pgx_ulid.control

```diff
@@ -1,5 +1,5 @@
 comment = 'ulid type and methods'
```

Comment:
Why are we renaming the extension?

Reply:
I'm only renaming the crate, the build will generate a package that will provide 100% compatibility with the old name, so it won't break any existing Postgres extension